include/circt/Dialect/Seq/SeqOps.td

```diff
@@ -239,7 +239,6 @@ def FirRegOp : SeqOp<"firreg",
   );
   let results = (outs AnyType:$data);
 
-  let skipDefaultBuilders = 1;
```

Comment:
Why is this necessary? Aren't you just using the second custom builder or am I missing something?

Reply:
That's what I was trying initially, but it turns out that the second custom builder _requires_ the reset and reset value to be non-null. So you'd have to call either the first or second builder depending on whether you want a reset or not. Having the default builder available feels like a good thing.